xwiki-rendering-api/src/main/java/org/xwiki/rendering/block/ListBLock.java

```diff
@@ -24,7 +24,9 @@
  *
  * @version $Id$
  * @since 1.5M2
+ * @deprecated Use {@link ListBlock} instead
```

Comment:
@deprecated since 8.1M1, use {@link ListBlock} instead

since 8.1M1 will be the next release in this branch